lib/ohai/mixin/ec2_metadata.rb

```diff
@@ -19,6 +19,7 @@
 
 require "net/http"
 require "socket"
+require "json"
```

Comment:
Please use FFIYajl for JSON handling; see https://github.com/chef/ohai/blob/master/lib/ohai/mixin/gce_metadata.rb#L71 (also this validates that we've got valid JSON)